index.html

```diff
@@ -1,117 +1,286 @@
-                        <span>
-                            <svg width="24" height="24" viewBox="0 0 24 24" color="reward-400"><path d="M12 19.18l-6.678 4.71a.595.595 0 0 1-.904-.285.603.603 0 0 1-.015-.361L6.6 15.04.217 9.74a.6.6 0 0 1 .356-1.063l8.09-.33L11.436.401A.6.6 0 0 1 11.999 0a.595.595 0 0 1 .564.402l2.774 7.943 8.09.331a.596.596 0 0 1 .542.409.604.604 0 0 1-.186.654L17.4 15.04l2.198 8.203a.603.603 0 0 1-.235.648.594.594 0 0 1-.685 0L12 19.18z"></path></svg>
-                        </span>
-                        <span>
-                            <svg width="24" height="24" viewBox="0 0 24 24" color="reward-400"><path d="M12 19.18l-6.678 4.71a.595.595 0 0 1-.904-.285.603.603 0 0 1-.015-.361L6.6 15.04.217 9.74a.6.6 0 0 1 .356-1.063l8.09-.33L11.436.401A.6.6 0 0 1 11.999 0a.595.595 0 0 1 .564.402l2.774 7.943 8.09.331a.596.596 0 0 1 .542.409.604.604 0 0 1-.186.654L17.4 15.04l2.198 8.203a.603.603 0 0 1-.235.648.594.594 0 0 1-.685 0L12 19.18z"></path></svg>
-                        </span>
-                        <span>
-                            <svg width="24" height="24" viewBox="0 0 24 24" color="reward-400"><path d="M12 19.18l-6.678 4.71a.595.595 0 0 1-.904-.285.603.603 0 0 1-.015-.361L6.6 15.04.217 9.74a.6.6 0 0 1 .356-1.063l8.09-.33L11.436.401A.6.6 0 0 1 11.999 0a.595.595 0 0 1 .564.402l2.774 7.943 8.09.331a.596.596 0 0 1 .542.409.604.604 0 0 1-.186.654L17.4 15.04l2.198 8.203a.603.603 0 0 1-.235.648.594.594 0 0 1-.685 0L12 19.18z"></path></svg>
-                        </span>
-                        <span>
-                            <svg width="24" height="24" viewBox="0 0 24 24" color="reward-400"><path d="M12 19.18l-6.678 4.71a.595.595 0 0 1-.904-.285.603.603 0 0 1-.015-.361L6.6 15.04.217 9.74a.6.6 0 0 1 .356-1.063l8.09-.33L11.436.401A.6.6 0 0 1 11.999 0a.595.595 0 0 1 .564.402l2.774 7.943 8.09.331a.596.596 0 0 1 .542.409.604.604 0 0 1-.186.654L17.4 15.04l2.198 8.203a.603.603 0 0 1-.235.648.594.594 0 0 1-.685 0L12 19.18z"></path></svg>
-                        </span>
-                    </div>
-                    <span>Best Recipe 2020</span>
-                    <span>This was SOOOOO yummy! One of my faves!</span>
-                    <span>- Maria A.</span>
-                </div>
-                <span>Pub-Style Shepherd’s Pie</span>
-                <span>with White Cheddar & Thyme Mashed Potatoes</span>
-                <div><span>50 min</span></div>
-            </div>
+      <figure class="recipe-card">
+        <img
+          src="https://img.hellofresh.com/c_fill,f_auto,fl_lossy,h_432,q_auto,w_768/hellofresh_s3/image/kikkoman-tropical-teriyaki-pork-bowls-e8103c70.jpg"
+          class="recipe-card-image"
+          style="max-width: 100%; height: auto"
+        />
+        <h2>Kikkoman® Tropical Teriyaki Pork Bowls</h2>
+        <span class="recipe-description"
+          >with Kiwi Salsa, Lime Crema & Jasmine Rice</span
+        >
+        <div class="prep-time"><span>35 min</span></div>
```

Comment:
you can add a div around these and give it some padding to take the text off the edges without affecting the image

Reply:
(and for all of the other recipes)